torch/_dynamo/guards.py

```diff
@@ -200,6 +233,9 @@ def __init__(
             scope = {"L" if local else "G": user_scope}
         else:
             scope = {"L" if local else "G": dict()}
+        # Builtins are globals from the perspective of CPython
+        # We do merge them into globals to make them visible to guards
+        # However, we currently offer no mechanism for guarding on builtins
         self.scope: Dict[str, Dict[str, object]] = scope
         self.scope["__builtins__"] = builtins.__dict__.copy()
         for (
```

Comment:
All I really wonder is why we use builtins as a container for `torch.package.package_importer._package_imported_modules`

Reply:
Originally added: https://github.com/pytorch/pytorch/pull/91821